libraries/shared/storage/backfill/storage_value_loader.go

```diff
@@ -53,80 +59,96 @@ func (r *StorageValueLoader) Run() error {
 	}
 
 	for _, header := range headers {
-		for address, chunkedKeys := range addressToKeys {
-			for _, keys := range chunkedKeys {
-				persistStorageErr := r.getAndPersistStorageValues(address, keys, header.BlockNumber, header.Hash)
-				if persistStorageErr != nil {
-					return persistStorageErr
-				}
-			}
+		persistStorageErr := r.getAndPersistStorageValues(header.BlockNumber, header.Hash)
+		if persistStorageErr != nil {
+			return persistStorageErr
 		}
+
 	}
-	logrus.Infof("Finished persisting storage values for %v addresses from block %v to %v.", len(addressToKeys), r.startingBlock, r.endingBlock)
+	logrus.Infof("Finished persisting storage values for %v addresses from block %v to %v.", len(r.addressesToKeysToValues), r.startingBlock, r.endingBlock)
 
 	return nil
 }
 
-func (r *StorageValueLoader) getStorageKeys() (map[common.Address][][]common.Hash, error) {
-	addressToKeys := make(map[common.Address][][]common.Hash, len(r.initializers))
+func (r *StorageValueLoader) setStorageKeys() error {
```

Comment:
`addressesToKeysToValues` isn't called `StorageKeys` anywhere so it takes some reading to realize what you mean by StorageKyes in the function name.

Reply:
Name change incoming 📬